### ISSUE TITLE
🐛 Failed to update ui::AXTree, error: 307 will not be in the tree and is not the new root

### ISSUE THREAD

**Issue by jonahzheng** [bdlukaa/fluent_ui#163]
```
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 51 is not marked for destruction, would be reparented to 3
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: Node 51 is not marked for destruction, would be reparented to 3
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: 307 will not be in the tree and is not the new root
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: 307 will not be in the tree and is not the new root
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: 307 will not be in the tree and is not the new root
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: 307 will not be in the tree and is not the new root
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: 307 will not be in the tree and is not the new root
[ERROR:accessibility_bridge.cc(76)] Failed to update ui::AXTree, error: 307 will not be in the tree and is not the new root
````

![failed](https://user-images.githubusercontent.com/5125282/152961841-12178c6c-5ca2-4b94-a566-1753f31b4278.gif)

```
D:\flutter\flutter doctor -v
[√] Flutter (Channel stable, 2.10.0, on Microsoft Windows [Version 10.0.19042.1466], locale zh-CN)
    • Flutter version 2.10.0 at D:\flutter_sdk
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5f105a6ca7 (7 days ago), 2022-02-01 14:15:42 -0800
    • Engine revision 776efd2034
    • Dart version 2.16.0
    • DevTools version 2.9.2

[√] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at C:\Android\SDK
    • Platform android-32, build-tools 32.0.0
    • ANDROID_HOME = C:\Android\SDK
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Professional 2019 16.11.8)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional
    • Visual Studio Professional 2019 version 16.11.32002.261
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] IntelliJ IDEA Ultimate Edition (version 2019.2)
    • IntelliJ at C:\Program Files\JetBrains\IntelliJ IDEA 2019.2.1
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart

[√] VS Code, 64-bit edition (version 1.63.2)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension can be installed from:
       https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19042.1466]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 97.0.4692.99
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 96.0.1054.62

[!] HTTP Host Availability
    X HTTP host https://maven.google.com/ is not reachable. Reason: An error occurred while checking the HTTP host: 

    X HTTP host https://cloud.google.com/ is not reachable. Reason: An error occurred while checking the HTTP host: 

! Doctor found issues in 1 category.

```

**Comment by bdlukaa** [comment]
This isn't an issue with the library, but with Flutter's Accessibility support for Windows.

**Comment by jonahzheng** [comment]
thank you, yes, it is only an warning,
my steps:
1、run https://github.com/bdlukaa/fluent_ui/tree/master/example;
2、click NavigationPane's search icon, open NavigationPane;
3、and close NavigationPane, will occus issue.

Flutter version 2.10.0

**Comment by jonahzheng** [comment]
flutter version switch back to 2.8.1,
the error is disappear
